### PR TITLE
Re-submission of PR with merge issues corrected

### PR DESCRIPTION
**Pull request by ARamsay17**
Improvements to uncertainty breakdown plots
Minor cleanup of PIU.py (cleanup will continue)

### CODE REVIEW

File: Source/ProcessL1aqc.py
[the remark this answers is not in the document]
Comment (oceancolorcoder) [reply Source/ProcessL1aqc.py:1]: @ARamsay17?

**Review by oceancolorcoder — CHANGES_REQUESTED** [review]
@ARamsay17, Please reply to the review question on PL1AQC. 

Also, please list how the changes were tested prior to the PR (sample pySAS? TriOS? L0-L2? Regime?)

File: Source/ProcessL1aqc.py
[the remark this answers is not in the document]
Comment (ARamsay17) [reply Source/ProcessL1aqc.py:1]: @oceancolorcoder I'm confused myself as I don't recall ever making this change. Please revert

**Comment by oceancolorcoder** [comment]
Closing in favor of a new PR to come.

**Pull request closed without merging**